src/components/ScheduleView.tsx

```diff
@@ -17,7 +17,7 @@ export interface ScheduleEntry {
   day: number
 }
 
-const PREVIEW_END = new Date('2025-09-10T15:30:00-07:00')
+const PREVIEW_END = new Date('2025-09-10T17:30:00-07:00')
```

Comment:
<img width="457" height="28" alt="image" src="https://github.com/user-attachments/assets/70980b1c-aa32-444d-a5a3-7a4330905771" />

I think the Preview date is 9/7, not 9/10 (which was just something I threw in as a guess earlier, I think).